include/deal.II/cgal/utilities.h

```diff
@@ -601,28 +601,24 @@ namespace CGALWrappers
    * @param mapping Mapping object for the cell.
    * @return  Array of vertices in CGAL order.
    */
-  template <int n_vertices, int dim, int spacedim>
-  std::array<Point<spacedim>, n_vertices>
+  template <int dim, int spacedim>
+  void
   get_vertices_in_cgal_order(
     const typename dealii::Triangulation<dim, spacedim>::cell_iterator &cell,
-    const Mapping<dim, spacedim> &                                      mapping)
+    const Mapping<dim, spacedim> &                                      mapping,
+    std::vector<Point<spacedim>> &                                      dst)
```

Comment:
You should also assert that `dst` has size `n_vertices`. Is there a reason not to return the result as it was previously done?

Reply:
This change was done by @fdrmrc. From my side, both is fine.

Reply:
No, there's no particular reason actually. I've just realized that if you want to use this version we need to update the documentation for this function. Sorry about that.

Reply:
My preference would be to return the result again but at the very least the documentation must match what the function is doing.

Reply:
So done